doc/sphinx/src/getting-started.rst

```diff
@@ -9,16 +9,35 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
 
   git clone --recursive git@github.com:lanl/singularity-eos.git
   cd singularity-eos
-  mkdir bin
-  cd bin
-  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
-  make -j
-  make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
+  cmake -B builddir -S . -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON 
```

Comment:
I think that this `buildidir` should be `build`? just like the ones on line 13 and 14. I saw there was a change in name and probably that one was forgotten.

Reply:
Correct, my mistake!